forage-android/src/main/java/com/joinforage/forage/android/network/data/CapturePaymentRepository.kt

```diff
@@ -93,27 +93,24 @@ internal class CapturePaymentRepository(
         var attempt = 1
 
         while (true) {
-            logger.debug("Polling capture payment message status. Attempt: $attempt.")
+            logger.i("Polling for balance check response for Payment $paymentRef")
 
             when (val response = messageStatusService.getStatus(contentId)) {
                 is ForageApiResponse.Success -> {
                     val paymentMessage = Message.ModelMapper.from(response.data)
 
                     if (paymentMessage.status == "completed") {
-                        logger.debug("Status is completed.")
                         if (paymentMessage.failed) {
-                            logger.debug("Failed is true.")
                             val error = paymentMessage.errors[0]
+                            logger.e("Received response ${error.statusCode} for capture request of Payment $paymentRef with message: ${error.message}")
                             return ForageApiResponse.Failure(listOf(ForageError(error.statusCode, error.forageCode, error.message)))
                         }
                         break
-                    } else {
-                        logger.debug("Status is ${paymentMessage.status}.")
                     }
 
                     if (paymentMessage.failed) {
-                        logger.debug("Failed is true.")
                         val error = paymentMessage.errors[0]
+                        logger.e("Received response ${error.statusCode} for capture request of Payment $paymentRef with message: ${error.message}")
```

Comment:
include `contentId` in log? Also, worth including `paymentRef` in log tag / Context for easier DataDog search?